### ISSUE TITLE
v6.1.3 - Unable to install any https plugin 403 Forbidden

### ISSUE THREAD

**Issue by texnofobix** [errbotio/errbot#1427]
### I am...

* [x] Reporting a bug
* [ ] Suggesting a new feature
* [ ] Requesting help with running my bot
* [ ] Requesting help writing plugins
* [ ] Here about something else

### I am running...

* Errbot version: 6.1.3
* OS version: Debian 10.3
* Python version: 3.7.3
* Using a virtual environment: yes

### Issue description
Unable to issue !repos install any plugin without error. 

errbot.log excerpt:
```log
2020-05-08 09:04:19,932 INFO     errbot.repo_manager       No repo index, creating it.
2020-05-08 09:04:20,261 ERROR    errbot.repo_manager       Could not update from source https://errbot.io/repos.json, keep the index as it is.
Traceback (most recent call last):
  File "/home/texnofobix/.errbot-ve/lib/python3.7/site-packages/errbot/repo_manager.py", line 157, in index_update
    with urlopen(url=source, timeout=10) as request:  # nosec
  File "/usr/lib/python3.7/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 563, in error
    result = self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 755, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/usr/lib/python3.7/urllib/request.py", line 531, in open
    response = meth(req, response)
  File "/usr/lib/python3.7/urllib/request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python3.7/urllib/request.py", line 569, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 503, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.7/urllib/request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 403: Forbidden
2020-05-08 09:04:20,262 ERROR    errbot.core               An error happened while processing a message ("!repos install https://errbot.readthedocs.io/en/latest/repos.json"): Traceback (most recent call last):
  File "/usr/lib/python3.7/shelve.py", line 111, in __getitem__
    value = self.cache[key]
KeyError: 'repo_index'
```
Based on testing, the Python-urllib User Agent is being denied 403 Forbidden. 

### Steps to reproduce

1. From virtualenv install instructions, `~/.errbot-ve/bin/errbot --init` and `~/.errbot-ve/bin/errbot`
2. Start errbot with `~/.errbot-ve/bin/errbot`
3. Issue  `!repos install https://github.com/sijis/err-kudos.git` for example.

### Additional info

If you have any more information, please specify it here.

Possible resolution/workaround to repo_manager.py:
```python
from .version import VERSION

    def index_update(self) -> None:
        index = {LAST_UPDATE: datetime.now().timestamp()}
        for source in reversed(self.plugin_indexes):
            try:
                if urlparse(source).scheme in ('http', 'https'):
                    request=Request(url=source, headers={ 'User-Agent': 'Errbot/{}}'.format(VERSION)})
                    with urlopen(request, timeout=10) as response:  # nosec
                        log.debug('Update from remote source %s...', source)
                        
                        encoding = response.headers.get_content_charset()
                        content = response.read().decode(encoding if encoding else 'utf-8')
                else:
                    with open(source, encoding='utf-8', mode='r') as src_file:
                        log.debug('Update from local source %s...', source)
                        content = src_file.read()
                index.update(json.loads(content))
            except (HTTPError, URLError, IOError):
                log.exception('Could not update from source %s, keep the index as it is.', source)
                break
        else:
            # nothing failed so ok, we can store the index.
            self[REPO_INDEX] = index
            log.debug('Stored %d repo entries.', len(index) - 1)
```

**Comment by sijis** [comment]
This should be fixed in master already.

Could you install from master and see if the issue is solved for you?

Thanks.

**Comment by texnofobix** [comment]
I did not look at master until now. Master branch works.  Not sure if you want to use the idea of adding the version to the UserAgent. 

Thanks!

**Comment by sijis** [comment]
@texnofobix Thanks for validating.
In this case I'm not sure adding the version makes a difference, however, thank you for the suggestion.

**Comment by sijis** [comment]
Should be fixed in the latest 6.1.4 release.
